As requested by PureStorage Support Team from MarkSymsCtx

From: Yann LE BRIS <yann.lebris@vates.tech>

More details: https://github.com/xapi-project/sm/pull/729#discussion_r1967525880

Signed-off-by: Yann LE BRIS <yann.lebris@vates.tech>

diff --git a/udev/99-purestorage.rules b/udev/99-purestorage.rules
index 49b4295..6194950 100644
--- a/udev/99-purestorage.rules
+++ b/udev/99-purestorage.rules
@@ -9,7 +9,5 @@ ACTION=="add|change", KERNEL=="dm-[0-9]*", SUBSYSTEM=="block", ENV{DM_NAME}=="36
 
 # Spread CPU load by redirecting completions to originating CPU
 ACTION=="add|change", KERNEL=="sd*[!0-9]", SUBSYSTEM=="block", ENV{ID_VENDOR}=="PURE", ATTR{queue/rq_affinity}="2"
-ACTION=="add|change", KERNEL=="dm-[0-9]*", SUBSYSTEM=="block", ENV{DM_NAME}=="3624a937*", ATTR{queue/rq_affinity}="2"
 
-# Set the HBA timeout to 60 seconds
-ACTION=="add", SUBSYSTEMS=="scsi", ATTRS{model}=="FlashArray      ", RUN+="/bin/sh -c 'echo 60 > /sys/$DEVPATH/device/timeout'"
+ACTION=="add", SUBSYSTEMS=="scsi", ATTRS{model}=="FlashArray      ", ATTR{device/timeout}="60"
\ No newline at end of file
